src/Http/Controllers/U2fController.php

```diff
@@ -35,90 +29,83 @@ public function __construct(LaravelU2f $u2f, Config $config)
     }
 
     /**
-     * @author LAHAXE Arnaud
```

Comment:
Feel free to change my commit or add his author name in an extra commit. On mobile atm for a few days. 

I was already wondering if you guys were planning to just merge it or I had to start my own fork & maintain it. I was working on recovery codes etc. 

Anyways feel free to change stuff on my github & add his name back.